### DISCUSSION TITLE
Allow multiple ranges

### DISCUSSION THREAD

**Discussion by TheChilliPL** [mistval/kotoba#198] [Ideas]
There's a range feature allowing to start a quiz like
```
k!q hiragana(1-10)
```
It would be awesome to be able to specify multiple ranges for each deck

Neither
```
k!q hiragana(1-5,10-15)
```
nor
```
k!q hiragana(1-5),hiragana(10-15)
```
work currently :/

**Comment by mistval** [comment]
👍 I would probably make it like `k!q hiragana(1-5)+hiragana(10-15)` so it would be the same as deck mixing.

**Comment by mistval** [comment]
This has now been implemented, in the way I mentioned above: `k!q hiragana(1-5)+hiragana(10-15)`
